tests/python/unittest/test_target_codegen_opencl.py

```diff
@@ -119,7 +119,27 @@ def check_max(dev, n, dtype):
     check_max(dev, 1, "float32")
     check_max(dev, 1, "float64")
 
+@tvm.testing.requires_gpu
+@tvm.testing.requires_opencl
```

Comment:
Thanks for adding this test! This is a bit of a meta-comment, but we are in the process of changing our testing infra to run fewer integration tests (e.g. running on device) to more unit tests during pre-commit. In this case, you could consider an additional unit test that removes the call to to run the kernel and replace it with a check for `erf` in the opencl source,

```
source_str = fun.imported_modules[0].get_source()
```

to ensure it is not `erff` as it is for cuda. 

The above test would be a unit test that can run in pre-commit with the full integration test (build + accuracy comparison) could be run as a nightly.

Reply:
Thank for the comment! I will replace it with getting opencl source and checking there is `erf` in it instead of `erff`.